azure-kusto-ingest/azure/kusto/ingest/descriptors.py

```diff
@@ -20,6 +22,10 @@ def assert_uuid4(maybe_uuid, error_message):
 class FileDescriptor:
     """FileDescriptor is used to describe a file that will be used as an ingestion source."""
 
+    # The compressed sized of a 4GB file that will not cause the gzip stream size (unsigned int) to rollover given a 1:40 compression factor
+    c_gzip_max_disk_size_for_detections = int(4 * 1024 * 1024 * 1024 / 40)
+    c_default_compression_factor = 11
```

Comment:
what does the c_ prefeix stand for?

Reply:
Const.
I know python does not have such a concept, but figured this would make one weary of making changes to the value.

Reply:
the convention is just upper case